resource_manager/filter_plugins/param_list_compare.py

```diff
@@ -0,0 +1,36 @@
+from ansible.errors import AnsibleFilterError
```

Comment:
Please add documentation for filter plugin ad pass it through argspec validator Refer https://github.com/ansible-collections/ansible.utils/blob/main/plugins/filter/validate.py#L5 and https://github.com/ansible-collections/ansible.utils/blob/main/plugins/filter/validate.py#L79